configs/sites/nautilus/packages.yaml

```diff
@@ -2,21 +2,44 @@ packages:
   all:
     compiler:: [intel@2021.5.0, aocc@4.0.0]
     providers:
-      mpi:: [intel-oneapi-mpi@2021.5.1, openmpi@4.1.4]
+      # For now need to enable one or the other;
+      # see https://github.com/JCSDA/spack-stack/issues/659
+      mpi:: [openmpi@4.1.5rc2]
+      #mpi:: [openmpi@4.1.4]
+      blas:: [intel-oneapi-mkl]
+      lapack:: [intel-oneapi-mkl]
+  ectrans:
+    variants:: +mkl
+  gsibec:
+    variants:: +mkl
 
 ### MPI, Python, MKL
   mpi:
     buildable: False
-  intel-oneapi-mpi:
-    externals:
-    - spec: intel-oneapi-mpi@2021.5.1%intel@2021.5.0
-      modules:
-      - intel/mpi/2021.5.1
-      prefix: /p/app/compilers/intel/oneapi
+  #intel-oneapi-mpi:
+  #  externals:
+  #  - spec: intel-oneapi-mpi@2021.5.1%intel@2021.5.0
+  #    modules:
+  #    - intel/mpi/2021.5.1
+  #    prefix: /p/app/compilers/intel/oneapi
```

Comment:
Should the commented code be deleted? Or is there a reason to keep it. Thanks

Reply:
Yes, this needs to be kept. I expect the sysadmins will fix the problem with oneAPI on the Mellanox fabric soon, and then NRL will want to switch back.

Reply:
Thanks for the explanation. I think it's fine to leave these in.